TipCatDotNet.Api/Services/Payments/PayoutService.cs

```diff
@@ -0,0 +1,94 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using Microsoft.EntityFrameworkCore;
+using Stripe;
+using TipCatDotNet.Api.Data;
+
+namespace TipCatDotNet.Api.Services.Payments;
+
+public class PayoutService : IPayoutService
+{
+    public PayoutService(AetherDbContext context, Stripe.PayoutService payoutService)
+    {
+        _context = context;
+        _payoutService = payoutService;
+    }
+
+
+    public Task<Result> PayoutAll(CancellationToken cancellationToken = default)
+    {
+        return Result.Success()
+            .Bind(PayoutByUsers);
+
+
+        async Task<Result> PayoutByUsers()
```

Comment:
You use two terms `PayoutAll` and `PayoutByUsers` for one action. According to our guidelines I suggest `Payout` and `PayoutInternal`